### PR TITLE
LPD-24891

### PR DESCRIPTION
**Pull request by icgsbr**
https://liferay.atlassian.net/browse/LPD-24891

> _WARNING: The test from ObjectForms#CanRetrieveDataProvidersOnSelectFromListField was failing on Master before my implementation_

### CODE REVIEW

File: build-test.xml [hunk 1]
```diff
@@ -13080,6 +13080,15 @@ captcha.enforce.disabled=true</echo>
 			</then>
 		</if>
 
+		<if>
+			<isset property="default.admin.password" />
```
Comment (carolmariaabb) [inline build-test.xml:13084]: @icgsbr please sort alphabetically, i.e., `default.admin.password` before the `digital.signature.account.base.uri` property.

**Review by carolmariaabb — CHANGES_REQUESTED** [review]

File: build-test.xml (hunk 1 above)
Comment (icgsbr) [reply build-test.xml]: Done

**Comment by icgsbr** [comment]
ci:test:sf

**Review by carolmariaabb — APPROVED** [review]

**Comment by carolmariaabb** [comment]
ci:test:object

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:forward:force

**Pull request closed without merging**